src/Entity/MeetupEvent.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\MeetupEventRepository;
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=MeetupEventRepository::class)
+ */
```

Comment:
(optional)
Quizas agregar que el meetupId sea unico 
```
 * @UniqueEntity("meetupId")
```

Reply:
It was done, using the unique constraint into the field to avoid an extra import